### PR TITLE
chore: Disable Prettier for auto-generated files

### PR DESCRIPTION
**Pull request by bennyn**

### CODE REVIEW

File: .prettierignore [hunk 1]
```diff
@@ -0,0 +1,2 @@
+/package.json
+/yarn.lock
```
Comment (lipis) [inline .prettierignore]: yarn.lock is not being formatted anyway by Prettie.. but I don't agree that `package.json` is autogenerated..

**Comment by lipis** [comment]
Why is that?!

**Comment by bennycode** [comment]
@lipis I removed `/package.json` from `.prettierignore`. Fine now?

File: .prettierignore [hunk 2]
```diff
@@ -0,0 +1 @@
...
```
Comment (lipis) [inline .prettierignore:1]: this file will never be prettified anyway.. it's not supported.

Comment (bennycode) [reply .prettierignore:1]: ok

**Comment by lipis** [comment]
sorry man.. yeah.. it's fine now! :P

**Pull request closed without merging**